src/utils/utils.js

```diff
@@ -194,7 +192,7 @@ export function processUrlQuery(src) {
 
 	try {
 		const res = {};
-		for (const [key, value] of new URL(src).searchParams.entries()) {
+		for (const [key, value] of new URL(src).searchParams.entries()) { // eslint-disable-line no-restricted-syntax
 			res[key] = value;
 		}
```

Comment:
Could we avoid having to suppress the linter if we did `res = { ... new URL(src).searchParams.entries() }` instead?

Reply:
It code didn't work in my tests.  Nor does it address the problem that the linter doesn't like the `entries()` function as it is an Iterator.

Reply:
Roger. Thanks for trying it.